src/encoded/schemas/fvp_b4.json

```diff
@@ -0,0 +1,243 @@
+                        "2 - Moderate Severely impaired in handling problems, similarities, and differences; social judgment usually impaired.",
+                        "3 - Severe Unable to make judgments or solve problems."
+                    ]
+                },
+                "fu_commun": {
+                    "title": "4. COMMUNITY AFFAIRS",
+                    "type": "string",
+                    "enum": [
+                        "0 - None Independent function at usual level in job, shopping, volunteer and social groups.",
+                        "0.5 - Questionable Slight impairment in these activities.",
+                        "1 - Mild Unable to function independently at these activities, although may still be engaged in some; appears normal to casual inspection.",
+                        "2 - Moderate No pretense of independent function outside the home; appears well enough to be taken to functions outside the family home.",
+                        "3 - Severe No pretense of independent function outside the home; appears too ill to be taken to functions outside the family home."
+                    ]
+                },
+                "fu_homehobb": {
+                    "title": "5. HOME & HOBBIES",
+                    "type": "string",
+                    "enum": [
+                        "0 - None Life at home, hobbies, and intellectual interests well maintained.",
+                        "0.5 - Questionable Life at home, hobbies, and intellectual interests slightly impaired.",
+                        "1 - Mild Mild but definite impairment of function at home; more difficult chores abandoned; more complicated hobbies and interests abandoned.",
+                        "2 - Moderate Only simple chores preserved; very restricted interests, poorly maintained.",
+                        "3 - Severe No significant function in the home."
+                    ]
+                },
+                "fu_perscare": {
+                    "title": "6. PERSONAL CARE",
+                    "type": "string",
+                    "enum": [
+                        "0 - None / Questionable - 0Fully capable of self-care (=0).",
+                        "1 - Mild Needs prompting.",
+                        "2 - Moderate Requires assistance in dressing, hygiene, keeping of personal effects.",
+                        "3 - Severe Requires much help with personal care; frequent incontinence."
+                    ]
+                },
+                "fu_cdrsum": {
+                    "title": "7. CDR SUM OF BOXES",
+                    "description": "Link to Global CDR calculation tool: https://www.alz.washington.edu/cdrnacc.html",
+                    "type": "number"
+                },
+                "fu_cdrglob": {
+                    "title": "8. GLOBAL CDR",
+                    "type": "string",
+                    "enum": [
+                        "0.0 = No Impairment",
```

Comment:
The keys for question 8 should be: 0, 0.5, 1, ,2 ,3.
![image](https://user-images.githubusercontent.com/44071821/100885825-b5751080-3478-11eb-90f3-adcef47d646d.png)

Reply:
I got confused of this one. If 0.5 can be resolved, how the 1.0, 2.0, 3.0 cannot?
 Can you write down what type you want to change in Redcap forms, so I can change relatively? @mingjiecn Thanks.